libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -422,6 +422,54 @@ void AP_Mount::handle_gimbal_manager_set_attitude(const mavlink_message_t &msg){
     }
 }
 
+void AP_Mount::handle_command_gimbal_manager_set_pitchyaw(const mavlink_message_t &msg){
+    mavlink_gimbal_manager_set_pitchyaw_t packet;
+    mavlink_msg_gimbal_manager_set_pitchyaw_decode(&msg,&packet);
+
+    AP_Mount_Backend *backend;
+
+    // check gimbal device id.  0 is primary, 1 is 1st gimbal, 2 is
+    // 2nd gimbal, etc
+    const uint8_t instance = packet.gimbal_device_id;
+    if (instance == 0) {
+        backend = get_primary();
+    } else {
+        backend = get_instance(instance - 1);
+    }
+
+    if (backend == nullptr) {
+        return;
+    }
```

Comment:
I have a suspicion we've duplicated this code several times now.

Something to fix up in a future PR.

Reply:
Noted